Formula/imapsync.rb

```diff
@@ -0,0 +1,108 @@
+class Imapsync < Formula
+  desc "Migrate or backup IMAP mail accounts"
+  homepage "http://ks.lamiral.info/imapsync/"
```

Comment:
homepage can be replaced with this secure one:
`https://imapsync.lamiral.info/`